tests/test_authz_authorizer.py

```diff
@@ -38,37 +32,161 @@ def test__init__(self):
         assert self.authz.resource == "test_resource"
         assert self.authz.permission == "permission_name"
 
-    def test__repr__(self):
+    def test__repr__(self) -> None:
         assert repr(self.authz) == (
             "Authorizer(auth_jwt=<jwt>, "
             "resource_name='test_resource', "
             "permission_name='permission_name')"
         )
 
-    def test__set_policy_w_scope(self):
+    def test_validate_one_with_expired(self) -> None:
+        expiration_negative = int((datetime.utcnow() - timedelta(seconds=1)).timestamp())
```

Comment:
I also see that some of the other comments from #40 were not applied - I don't see any point in repeating them here, but I'd like to see the fixes 😉